packages/compass-web/src/connection-storage.tsx

```diff
@@ -255,15 +277,62 @@ export class AtlasCloudConnectionStorage
     });
   }
 
-  private async _loadAndNormalizeClusterDescriptionInfo(): Promise<
+  private async _loadAndNormalizeClusterDescriptionInfoV2(): Promise<
+    ConnectionInfo[]
+  > {
+    return this.atlasService
+      .authenticatedFetch(
+        this.atlasService.cloudEndpoint(
+          `/explorer/v1/groups/${this.projectId}/clusters/connectionInfo`
+        )
+      )
+      .then((res) => {
```

Comment:
Is there a reason we're handling these promises with .then rather than await? I find await to be easier to read and error handle.

Reply:
No good reason not to use async / await, I'll switch, thanks for spotting!